jawampa-core/src/main/java/ws/wamp/jawampa/WampClient.java

```diff
@@ -333,7 +335,7 @@ public void run() {
      * of the call be Observable&lt;String&gt;.
      * @return An observable that can be used to subscribe on the topic.
      */
-    public <T> Observable<T> makeSubscription(final String topic, final Class<T> eventClass) {
+    public <T> Observable<MessageInfo<T>> makeSubscription(final String topic, final Class<T> eventClass) {
```

Comment:
Don't see a need to add makeSubscriptionWithDetails for this version because this is not prefix or wildcard subscriptions 
 public <T> Observable<T> makeSubscription(final String topic, final Class<T> eventClass) {
        return makeSubscription(topic, SubscriptionFlags.Exact, eventClass);
    }